src/test/java/com/marklogic/spark/reader/file/ReadArchiveFileTest.java

```diff
@@ -0,0 +1,41 @@
+package com.marklogic.spark.reader.file;
+
+import com.marklogic.spark.AbstractIntegrationTest;
+import com.marklogic.spark.Options;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.junit.jupiter.api.Test;
+
+import java.io.UnsupportedEncodingException;
+import java.util.HashSet;
+import java.util.List;
+
+public class ReadArchiveFileTest extends AbstractIntegrationTest {
+
+    @Test
+    void readArchiveFile() throws UnsupportedEncodingException {
+        Dataset<Row> reader = newSparkSession().read()
+            .format(CONNECTOR_IDENTIFIER)
+            .option(Options.READ_FILES_TYPE, "archive")
+            .load("src/test/resources/mlcp-archive-files/sample-XML.zip");
```

Comment:
This is loading an MLCP archive, but for MLE-12387, we want to read an NT archive file. Best way to do that is to reuse the test code from `WriteArchiveTest`. Can just copy and paste from there for now - e.g. I did a quick test like this:

```
@BeforeEach
    void beforeEach() {
        TestUtil.insertTwoDocumentsWithAllMetadata(getDatabaseClient());
    }

    @Test
    void test(@TempDir Path tempDir) {
        newSparkSession().read().format(CONNECTOR_IDENTIFIER)
            .option(Options.CLIENT_URI, makeClientUri())
            .option(Options.READ_DOCUMENTS_COLLECTIONS, "collection1")
            .option(Options.READ_DOCUMENTS_CATEGORIES, "content,metadata")
            .load()
            .repartition(1)
            .write().format(CONNECTOR_IDENTIFIER)
            .option(Options.WRITE_FILES_COMPRESSION, "zip")
            .mode(SaveMode.Append)
            .save(tempDir.toFile().getAbsolutePath());

        sparkSession.read().format(CONNECTOR_IDENTIFIER)
            .option(Options.READ_FILES_TYPE, "archive")
            .load(tempDir.toFile().getAbsolutePath())
            .show();
    }
```